LLVM_ASAN/Using-Address-Sanitizer-with-a-Short-HIP-Application.md

```diff
@@ -75,29 +74,24 @@ $ ldd mini
 This confirms that the address sanitizer runtime is linked in, and the ASAN instrumented version of the runtime libraries are used.
 Checking the `PATH` yields
 
-```
-
+```shell
 $ which llvm-symbolizer
 /opt/rocm-5.7.0-99999/llvm/bin/llvm-symbolizer
-
 ```
 
 Lastly, a check of the OS kernel version yields
 
-```
-
+```shell
 $ uname -rv
 5.15.0-73-generic #80~20.04.1-Ubuntu SMP Wed May 17 14:58:14 UTC 2023
-
 ```
 
 which indicates that the required HMM support (kernel version > 5.6) is available.
 This completes the necessary setup.
 Running with `m = 100`, `n1 = 11`, `n2 = 10` and `c = 100` should produce
 a report for an invalid access by the last 10 threads.
 
-```
-
+```gdb
```

Comment:
I don't think `gdb` is a valid code-block language. It might be better to just use `text` in these cases.